src/qml/components/UploadProgressBar.qml

```diff
@@ -0,0 +1,115 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import "../theme"
+
+
+Rectangle {
+    id: uploadProgressBar
+
+    property var model: models.fileCloudUploader
+    property int fileIndex: model.currentIndex
+    property int filesCount: model.fileNames.length
+    property real fileUploadValue: model.currentProcessedBytes * 100 / model.currentTotalBytes
+
+    property real uploadHeight: 60
+    property real uploadOpacity: 1
+
+    property real uploadFrom: 0
+    property real uploadTo: 100
```

Comment:
Use model.currentTotalBytes for **to** and model.currentProcessedBytes for **value**
0..100 range invokes extra conversion

Reply:
Yes. That is reasonable